README.md

```diff
@@ -71,6 +71,23 @@ Then the following data is inserted in the store:
         as:deleted "thetimeofdelete"^^xsd:dateTime  .
 ```
 
+## Tracking history
+
+Form definitions can specify that their history should be tracked by adding a triple `<formDefinitionUri> ext:withHistory "true"^^xsd:boolean`. When an instance of such a form is created or updated, the current value for that instance is written to a **new graph** with an auto-generated uri, e.g. `<http://mu.semte.ch/vocabularies/ext/formHistory/d420c907-8bbd-4f5b-b89c-be330df247ea>`. Meta information about this history item is written to a specific graph `<http://mu.semte.ch/graphs/formHistory>`. This meta information is made up of the following triples:
```

Comment:
Isn't it a bit much to generate a new graph for each form instance? Couldn't we just put them in a graph per type or something like that (maybe based on the provided prefix)? Don't know if it makes much difference from a technical standpoint, but I feel it is just making the database more complex without any real benefit I can come up with. While I'm typing this I can see that you could just fetch everything in this graph when you fetch the history, but that feels wrong.

Reply:
This part also mentions that the current value of the instance is written to a new graph, this is a bit vague to me. What exactly is written to this graph, the complete instance like it was before you changed it? And won't this result in problems when you have two objects with different data? But of course this all depends on access rights etc. but I could see some confusion or possible problems here.

Reply:
First point is clarified, and clear now, second point still requires a small rewrite.